sdks/python/gen_protos.py

```diff
@@ -123,7 +123,19 @@ def generate_urn_files(out_dir, api_path):
   This is executed at build time rather than dynamically on import to ensure
   that it is compatible with static type checkers like mypy.
   """
-  from google._upb import _message
+  try:
+    from google._upb import _message
+    list_types = (
+        list,
+        _message.RepeatedScalarContainer,
+        _message.RepeatedCompositeContainer,
+    )  # pylint: disable=c-extension-no-member
+  except ImportError:
+    from google.protobuf.internal import containers
+    list_types = (
+        list,
+        containers.RepeatedScalarFieldContainer,
+        containers.RepeatedCompositeFieldContainer)
```

Comment:
Trying to follow why we need this - is this from having an old protobuf installed? It looks like we previously used `google.protobuf.pyext._message` before https://github.com/apache/beam/commit/5cb1711c39c2ebc3005363f938ca4a4eee774333, is there a reason we're now using `google.protobuf.internal.containers`?

cc/ @AnandInguva @tvalentyn

Reply:
I think even with older protobuf, we would need `google.protobuf.pyext._message` when `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION==cpp` but we might need `google.protobuf.internal.containers` when `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python`. 

commented at https://github.com/apache/beam/pull/25914#discussion_r1144970641

Reply:
Sorry, this was part of another PR. That's been merged; rebased this PR.